commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleDrive.scala

```diff
@@ -8,55 +8,84 @@ import squants.motion.AngularVelocity
 import squants.{Acceleration, Dimensionless, Length, Percent, Velocity}
 import GenericValue._
 
+trait UnicycleProperties {
+  val maxForwardVelocity: Velocity
+  val maxTurnVelocity: AngularVelocity
+
+  val forwardControlGains: PIDFProperUnitsConfig[Velocity, Acceleration, Length, Dimensionless]
```

Comment:
All we're saying is that we want to store the PIDf gains for controlling the forward/backward speed, where the signal type is `Velocity`, the derivative type is `Acceleration`, the integral type is `Length`, and the output type is `Dimensionless`.

We really only take these gains in the case of some weird type of drivetrain (like an actual unicycle!) where we want to control forward/backward speed at the unicycle level. But in most cases this control will be disabled by the drivetrain trait that extends it.